src/main/java/frc/trigon/robot/subsystems/turret/TurretCommands.java

```diff
@@ -0,0 +1,22 @@
+package frc.trigon.robot.subsystems.turret;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+
+import java.util.function.Supplier;
+
+public class TurretCommands {
+    private static Turret TURRET = Turret.getInstance();
+
+    public static Command getSetMotorVoltageCommand(Supplier<Pose2d> robotPose) {
```

Comment:
this command will set the motor's voltage, yes. But what is its goal? why does it do that? give this a better name.